client/components/web-preview/toolbar.jsx

```diff
@@ -121,7 +121,7 @@ class PreviewToolbar extends Component {
 				{ showUrl &&
 					<ClipboardButtonInput
 						className="web-preview__url-clipboard-input"
-						value={ externalUrl || previewUrl }
+						value={ decodeURIComponent( externalUrl || previewUrl ) }
```

Comment:
`decodeURIComponent` [throws](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURIComponent#Exceptions) on bad input, so this should be wrapped in a try / catch